### PR TITLE
Add skeleton for November 14th release notes

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.4/release-3-4-5.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+### Update Software
+
+Once the new repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+<span class="twiki-macro NOTE"></span> Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-5.md]: This should be a `!!! note`

File: docs/release/3.4/release-3-4-5.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+<span class="twiki-macro NOTE"></span> Watch the yum update carefully for any messages about a `.rpmnew` file being created. That means that a configuration file had been editted, and a new default version was to be installed. In that case, RPM does not overwrite the editted configuration file but instead installs the new version with a `.rpmnew` extension. You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version into place (that is, without the `.rpmnew` extension). Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-5.md]: Also should be `!!! note`

**Comment by timtheisen** [comment]
Two people adding content. Need to start with a skeleton. In the future, the skeleton will be put up days before the release.

File: docs/release/3.4/release-3-4-5.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+### Update Software
+
+Once the new repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note
+Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-5.md]: This and the next `!!! note` formatting is wrong (subsequent lines need to be preceded by 4 spaces), see previous `!!! note`s

**Pull request merged**